src/manager/layout.js

```diff
@@ -156,8 +155,8 @@ export function unselectDocument(document) {
 }
 
 // Dialogs
-export function openDispatchAddon() {
-  layout.addonDispatchOpen = true;
+export function openDispatchAddon(addonId) {
+  layout.addonDispatchOpen = addonId;
```

Comment:
Not sure if this is the best way to handle and pass the plugin/addonId from the actionMenu into the dispatch API.

Reply:
I think this seems fine